### ISSUE TITLE
Proposal: Sync docs between lernajs.io and README.md

### ISSUE THREAD

**Issue by jasonxia23** [lerna/lerna#1871]
## Expected Behavior

Sync docs between lernajs.io and README.md to keep away contradiction.

## Current Behavior

README.md
> The instructions below are for Lerna 3.x. We recommend using it instead of 2.x for a new Lerna project.

lernajs.io
> Lerna 2.x is the recommended version to start with.

## Possible Solution

Generate lernajs.io from README.md, so we can keep them synchronized.

## Steps to Reproduce (for bugs)

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | VERSION |
| `npm --version`  | VERSION |
| `yarn --version` | VERSION |
| `node --version` | VERSION |

| OS | Version |
| --- | --- |
| NAME | VERSION |

**Comment by evocateur** [comment]
Yeah, this has been a long-term disjunction for the project. I hardly ever look at the website repo, and _ideally_ it would be some sort of Gatsby-based static website generated from a Travis hook of _this_ repo.

**Comment by jasonxia23** [comment]
Can we just add a deprecation warning in the website.
